db/seeds.rb

```diff
@@ -33,7 +33,9 @@
   caregiver.save
   puts "created caregiver named #{caregiver.first_name} with id #{caregiver.id}"
 
-  CITIES = %w(Asakusa Shibuya Minato Aoyama Giza Harajuku Kinshicho Nagatacho)
+  CITIES = %w(Asakusa Shibuya Minato Aoyama Ginza Harajuku Kinshicho Nagatacho)
+  TAGS = ['cooking', 'french', 'japanese', 'korean', 'guitar', 'painting', 'craft' '20 years playing tennis', 'outdoors', 'humor', 'globe trotter']
```

Comment:
@zuccamia Would tags be automatically saved to this TAGS array if a user put it as a skill section? I wonder how it works.